src/components/ViewContracts/form.js

```diff
@@ -19,65 +19,80 @@ const FormikEnhancer = withFormik({
   mapPropsToValues: () => {
     return ({
       fullName: '',
-      selectedUserId: '-1',
+      selectedEmployeeId: '-1',
+      selectedTeamId: '-1',
     });
   },
 
-  // Custom sync validation
-  validate: ({ fullName }) => {
-    let errors = {};
-    let nameArray = fullName.split(' ');
-    if (fullName === '') {
-      errors.fullName = 'First name cannot be empty.';
-    }
-    if (nameArray.length !== 2) {
-      errors.fullName = 'Please add both a first and last name';
-    }
-    return errors;
-  },
-
   handleSubmit: (payload, { props }) => {
-    props.searchUsers(payload);
+    props.searchEmployees(payload);
   },
 
 });
 
 
 export const SearchUserForm = props => {
-  const { users, isValid, setFieldValue, contractSearch } = props;
-  
-  const updateContract = (event) => {
-    const value = event.target.value;
-    contractSearch(event.target.value);
-    setFieldValue('selectedUserId', value);
+  const { employees, teams, isValid, setFieldValue, contractSearch } = props;
+
+  const updateContract = ({ target: { name, value } }) => {
+    const testIfValueIsANumber = parseInt(value);
+    if (!isNaN(testIfValueIsANumber)) {
+      contractSearch(name, value);
+      setFieldValue(name, value);
+    }
+  };
+
+  const createEmployeeLabelText = () => {
+    let title = 'Search for en employee in the text field above';
```

Comment:
Going forward these would be in constant files rather than strings in the files. One centralised folder to manage text.